lib/Pill/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+
+const Pill = props => {
+  return (
+    <span className={`pill ${props.className}`}>
```

Comment:
Should we be deconstruction the required props and then spread the rest onto here?

--------

This would allow us to do a thing like add DOM attr when required or even test attributes for react testing lib.

Reply:
Agree, that's done